programs/bpf_loader/Cargo.toml

```diff
@@ -25,6 +25,7 @@ thiserror = "1.0"
 [dev-dependencies]
 rand = "0.7.3"
 solana-runtime = { path = "../../runtime", version = "=1.11.4" }
+static_assertions = "1.1.0"
```

Comment:
why is this necessary in this pr, can we break it out into a separate pr?

Reply:
Not necessary, removed